main/webapp/modules/core/scripts/views/data-table/add-column-dialog.html

```diff
@@ -24,5 +25,6 @@
       <button class="button" bind="okButton"></button>
       <button class="button" bind="cancelButton"></button>
     </div>
+  </form>
```

Comment:
Thanks a lot! I wonder if wie need a 'type="submit"' in the "OK" button to keep it working?

Reply:
It is working, but I added the type="submit" to the ok button since that is standard.

Reply:
Interesting, do you understand why it was working without? Does the browser interpret the "OK" label by itself, to understand that the form should be submitted? I find that surprising!

Reply:
I checked and it looks like the type defaults to submit if inside a form.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type